cypress/integration/Login Test/loginTests.js

```
@@ -0,0 +1,70 @@
+/* === Test Created with Cypress Studio === */
+it('login test', function () {
+    /* ==== Generated with Cypress Studio ==== */
+    cy.visit('https://devapp.fleethawks.com/#/Login');
+    cy.get(':nth-child(1) > .input-group > .form-control').clear();
```

describe (Login Tests):
it ( should allow user to login when valid credentails are provided
it( should fail with errormessage when invalid credentials are provided)
it(should give validation error message when either username or password is not provided